Lib/getpass.py

```diff
@@ -156,17 +156,25 @@ def getuser():
 
     First try various environment variables, then the password
     database.  This works on Windows as long as USERNAME is set.
+    Any failure to find a username raises OSError.
 
+    .. versionchanged:: 3.13
```

Comment:
Move this into the getpass.rst docs for the function.

For the docstring, just keep it simple and mention in the new sentence that older Python versions raised a variety of exceptions instead.

Reply:
Thanks, I like that. If you prefer I preserve some of this detail in getpass.rst I could, but based on the other comment I just simplified in both places.

Ready for another look!